docker/buildroot.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+set -e
+
+docker_image="goranche/raspbian-ua-netinst"
+
+if [ "${1}" != "IN_DOCKER" ]; then
+	docker run --privileged -ti --rm -v $(pwd):/raspbian-ua-netinst "${docker_image}"
+	exit
+fi
+
```

Comment:
it creates a container from the image `${docker_image}`, and runs it...
the container will be removed after it's done (but not the image, that's something different altogether).
the current directory will be mapped to `/raspbian-ua-netinst` inside the container (that's how the resulting files end up in the same directory you ran it from)

this is pretty standard docker stuff, so it's documentation has tons of info on this 👍 

what did you mean by `IN_DOCKER`, though? 😊